src/core/atomic.d

```diff
@@ -181,6 +181,8 @@ else version( LDC )
     HeadUnshared!(T) atomicOp(string op, T, V1)( ref shared T val, V1 mod )
         if( __traits( compiles, mixin( "*cast(T*)&val" ~ op ~ "mod" ) ) )
     {
+        pragma(inline, true);
```

Comment:
It'd be nicer as function prefix (`pragma(inline, true) T func()`), right? Does that work correctly too? ;)